backend/core/package.json

```diff
@@ -130,5 +130,6 @@
   "engines": {
     "node": "12",
     "yarn": "^1.22"
-  }
+  },
+  "gitHead": "5f7694e3f2baff2a1a7a22e0676e487dda781a3e"
```

Comment:
I'm not sure this was intentionally added upstream. I found [this commit](https://github.com/bloom-housing/bloom/commit/0cbc5401ea9fd1d89623c499b3ed1f27bee8f98f) via blame, but there wasn't a PR linked to it for some reason.

I couldn't find much documentation about this issue, but I did find [some people complaining](https://github.com/lerna/lerna/issues/1880) about this happening accidentally with lerna, which Bloom uses.

Reply:
Done!